### PR TITLE
Storybook sample code fix

### PR DESCRIPTION
**Pull request by bluekurinji**

### CODE REVIEW

File: apps/angular-demo/src/app/text-area/text-area.component.ts [hunk 1]
```diff
@@ -7,9 +7,22 @@ import { Component } from '@angular/core';
 })
 export class TextAreaComponent {
 
+  value2 = "";
```
Comment (bluekurinji) [inline apps/angular-demo/src/app/text-area/text-area.component.ts:10]: Use proper naming for variable and events

File: libs/docs/src/components/common/Button.stories.mdx [hunk 2]
```diff
@@ -309,19 +309,19 @@ There are two button sizes: Normal (default) and Compact.
     <CodeSnippet
       code={`
         <goa-button-group alignment="start">
-          <goa-button leadingicon="airplane" (_click)="onClick" type="primary">Leading</goa-button>
-          <goa-button trailingIcon="airplane" (_click)="onClick" type="primary">Trailing</goa-button>
-          <goa-button leadingicon="archive" trailingicon="airplane" (_click)="onClick" type="primary">Both</goa-button>
+          <goa-button leadingicon="airplane" (_click)="onClick()" type="primary">Leading</goa-button>
+          <goa-button trailingIcon="airplane" (_click)="onClick()" type="primary">Trailing</goa-button>
```
Comment (bluekurinji) [inline libs/docs/src/components/common/Button.stories.mdx:313]: change trailingIcon to trailingicon

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**